source/_integrations/recorder.markdown

```diff
@@ -259,6 +259,8 @@ When using SQLite, if the system encounters unrecoverable disk corruption, it wi
 
 In this event, it may be possible to recover the old database by following the [SQLite recovery guide](https://www.sqlite.org/recovery.html).
 
+To ensure users are promptly informed of such incidents, the system will create a [persistent notification](/integrations/persistent_notification/) and will fire the event `recorder_database_corrupt` if corruption is detected. You can use this event in your [automations](/docs/automation/trigger/#event-trigger) to perform custom actions when corruption is detected.
```

Comment:
This seems incorrect, the event will befired even before the automation engine has started, so it would be impossible to do anything useful with it.

../Frenck